test/integration/fixtures/server12/mocks/dummyservice.js

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2015-2016 PointSource, LLC.
+ * MIT Licensed
+ */
+exports.init = function(logger) {
+    logger.info('Dummy Service Mock initialized');
+    process.exit(0); // exit test here
```

Comment:
Yeah that sounds much better. I might run into challenges but I'll do some exploration.